ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -100,35 +101,39 @@ export const WorkflowNodeSummary = (props: Props) => {
             value: <ResourcesDuration resourcesDuration={props.node.resourcesDuration} />
         });
     }
+    const showLogs = (container = 'main') => props.onShowContainerLogs(props.node.id, container);
     return (
         <div className='white-box'>
             <div className='white-box__details'>{<AttributeRows attributes={attributes} />}</div>
             <div>
-                <button className='argo-button argo-button--base-o' onClick={() => props.onShowYaml && props.onShowYaml(props.node.id)}>
```

Comment:
had to make them all grey buttons because the `-o` style button does not have a background fill, so I could not use it for a drop-down item